examples/sprite_animation/main.rs

```diff
@@ -102,31 +135,28 @@ impl SimpleState for Example {
                                 1.0,
                                 AnimationCommand::Start,
                             );
+                            self.animated = true;
                         }
-                    },
-                );
-                // All data loaded
-                self.progress_counter = None;
+                    }
+                }
             }
         }
+        buffer.flush(data.world);
+
         Trans::None
     }
 }
 
-fn initialise_camera(world: &mut World) {
+fn initialise_camera(world: &mut World, resources: &mut Resources) {
     let (width, height) = {
-        let dim = world.read_resource::<ScreenDimensions>();
+        let dim = resources.get::<ScreenDimensions>().unwrap();
```

Comment:
Probably use an except here. Helps new users if they copy code from the examples.